oio/account/backend.py

```diff
@@ -255,22 +255,28 @@ def update_container(self, account_id, name, mtime, dtime, object_count,
                 record[r] = 0
 
         ct = {name: 0}
-        pipeline = conn.pipeline(True)
-        pipeline.hmset(AccountBackend.ckey(account_id, name), record)
-        if deleted:
-            pipeline.expire(AccountBackend.ckey(account_id, name), EXPIRE_TIME)
-            pipeline.zrem('containers:%s' % account_id, name)
-        else:
-            pipeline.persist(AccountBackend.ckey(account_id, name))
-            pipeline.zadd('containers:%s' % account_id, **ct)
-        if incr_object_count:
-            pipeline.hincrby('account:%s' % account_id, 'objects',
-                             incr_object_count)
-        if incr_bytes_used:
-            pipeline.hincrby('account:%s' % account_id, 'bytes',
-                             incr_bytes_used)
-        pipeline.execute()
-        release_lock(conn, AccountBackend.ckey(account_id, name), lock)
+        try:
+            pipeline = conn.pipeline(True)
+            pipeline.hmset(AccountBackend.ckey(account_id, name), record)
+            if deleted:
+                pipeline.expire(AccountBackend.ckey(account_id, name),
+                                EXPIRE_TIME)
+                pipeline.zrem('containers:%s' % account_id, name)
+            else:
+                pipeline.persist(AccountBackend.ckey(account_id, name))
+                pipeline.zadd('containers:%s' % account_id, **ct)
+            if incr_object_count:
+                pipeline.hincrby('account:%s' % account_id, 'objects',
+                                 incr_object_count)
+            if incr_bytes_used:
+                pipeline.hincrby('account:%s' % account_id, 'bytes',
+                                 incr_bytes_used)
+            pipeline.execute()
+        except Exception:
+            name = None
```

Comment:
I don't think silencing the exception is a good thing. Just get rid of these two lines, or raise another exception with a known type. The return value of this method is never checked.